### PR TITLE
APPDEV-1265 fabric crash on build 26(206) in huawei device

### PR DESCRIPTION
**Pull request by Sivateja-CMI**
-- Added check to verify if app can perform given intent.
-- Nothing much can be done as OS does not support the given intent.

### CODE REVIEW

File: app/src/main/res/values-nl/strings.xml [hunk 1]
```diff
@@ -228,4 +228,5 @@
     <string name="yona_service_notification_channel_name">Vaste melding</string>
     <string name="support_mail_subject">Probleem met Yona app</string>
     <string name="mandatory_user_fetch_failure">De Yona-app heeft deze keer een netwerkverbinding nodig om te starten.</string>
+    <string name="usage_access_unavailable_message">Sta Yona toe om de gebruiksstatistieken van apparaten te controleren in apparaatinstellingen</string>
```
Comment (Bert-R) [inline app/src/main/res/values-nl/strings.xml]: Please make this:
> Sta Yona in het instellingenmenu toe om de gebruiksstatistieken van apps in te zien

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -230,4 +230,5 @@
     <string name="yona_service_notification_channel_name">Persistent notifications</string>
     <string name="support_mail_subject">Problem with Yona app</string>
     <string name="mandatory_user_fetch_failure">The Yona app this time needs network connectivity to start</string>
+    <string name="usage_access_unavailable_message">Please permit Yona to monitor devices usage statistics in device settings</string>
```
Comment (Bert-R) [inline app/src/main/res/values/strings.xml]: Please make this:
> In the device settings, please permit Yona to see app usage statistics

**Comment by Sivateja-CMI** [comment]
Closing this as the PR is raised against develop branch instead of master.
Raised a new [PR-585](https://github.com/yonadev/yona-app-android/pull/585)

**Comment by Bert-R** [comment]
This is merged through the other PR, right?

**Comment by Sivateja-CMI** [comment]
> This is merged through the other PR, right?

Yes bert.

**Pull request closed without merging**